Model/Order/Item/Builder.php

```diff
@@ -165,7 +165,11 @@ public function buildItemProductId(Item $item)
                 return $parentIds[0];
             }
         }
-        return (string)$item->getProductId();
+        $productId = (string)$item->getProductId();
```

Comment:
Why do we have to cast this to string? The method returns `int|null`

Reply:
We don't, trim removed. Thanks